src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -299,6 +299,11 @@ private static void copyResources(
         dstr = resource.destination;
       }
       if (dstr.charAt(0) == '/') dstr = dstr.substring(1);
+      if (dstr.startsWith("../")) {
+        // external repositories are positioned with a leading ../
+        // rename the path to external
+        dstr = "external" + dstr.substring(2);
```

Comment:
mind documenting this magic a bit more thoroughly? I don't follow this stripping of prefixes business

Reply:
What do you suggest we put here? I am just doing something that will work. We are coping files based on the bazel short path (which I think may now be the same as the path for these cases anyway). For a reason I don’t know, the path has ../ in it for external repos. As the code and comments explain, I am removing that so we don’t copy out of the directory we are working in.

Can you suggest a rewording of the comment that would have helped you?

Reply:
WDYT about:
```
       // paths to external repositories, for some reason, start with a leading ../
       // on the filesystem external repositories are positioned under external/
```
Additionally I'd extract this block to a method which can be named something like:
`renamePathToExternalRepository`